index.html

```diff
@@ -331,7 +331,108 @@ <h3 class="h4">AI tool usage</h3>
       </section>
 
       <!-- FAQS SECTION -->
-      <section id="faqs"></section>
+      <section id="faqs">
```

Comment:
Comparing to the designs, I think the FAQs section should have the same max width as the rules and registration sections. We can re-use the class I created for that here I think? `class="max-width-686"`